instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+
+
+class CommentSection extends React.Component {
+    constructor (props) {
+        super(props)
+        this.state = {
+            commentArray: props.commentInfo,
+            commentInput: ''
+        }  
+      }
+
+    
+
+      changeHandler = event => {
+        this.setState({[event.target.name]: event.target.value });
+      }
+
+      addNewComment = (event, index) => {
+          
+        event.preventDefault();
+        if (this.state.commentInput === '') {
```

Comment:
`.trim()` might accomplish the same effect but prevent also blank inputs. The user at this point is able to add a comment that is blank if he hits spacebar